### PR TITLE
Added youtube and slideshare links to post conference state

### PR DESCRIPTION
**Pull request by grahammendick**

### CODE REVIEW

File: shared/components/ConferenceHightlights/index.js [hunk 1]
```diff
@@ -27,50 +27,46 @@ const data = {
...
-    // },
-    // {
-    //   href: 'https://www.youtube.com/',
-    //   src: 'https://res.cloudinary.com/red-badger-assets/image/upload/v1486054146/react-london-2017/Ken_Wheeler_small.jpg.jpg', // eslint-disable-line
-    //   name: 'Ken Wheeler',
-    //   company: 'Formidable Labs',
-    //   title: 'Lightning talk: Offline for the greater good',
-    // },
+    {
+      href: 'https://www.youtube.com/watch?v=ziAShzxAVKY&t=12s',
```
Comment (dstodolny) [inline shared/components/ConferenceHightlights/index.js]: do we want to start that video at 12s?

Comment (grahammendick) [inline shared/components/ConferenceHightlights/index.js]: good question - i wondered the same. Amy gave me the link - it's the same one that's on the [playlist](https://www.youtube.com/channel/UCV4LIEkC0S9KUAPDm2g4mNQ)

Comment (gregdorward) [inline shared/components/ConferenceHightlights/index.js]: I agree, it looks like a mistake from when the link was originally copied and pasted. Other than that though, this is looking good

Comment (gregdorward) [inline shared/components/ConferenceHightlights/index.js]: I'm asking Amy about this

Comment (gregdorward) [inline shared/components/ConferenceHightlights/index.js]: Can we remove the =12s bit from the url please @grahammendick ?

Comment (grahammendick) [inline shared/components/ConferenceHightlights/index.js]: @gregdorward sure thing

**Comment by gregdorward** [comment]
I'm happy with this one to be merged once it's had a review, i've tested it across multiple viewports, browsers and devices, both js and non-js and it's all looking good

**Comment by gregdorward** [comment]
All good from me

**Pull request merged**